autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/SchedulePreview.css

```diff
@@ -40,16 +40,10 @@
 
 .color-box {
     display: inline-block;
-    border: 1px solid black;
+    outline: 1px solid black;
+    border: 1px solid white;
     margin-right: 4px;
-    padding: 1px;
-    width: 14px;
-    height: 14px;
-}
-
-.color-box span {
-    display: inline-block;
-    width: 100%;
-    height: 100%;
-    margin-bottom: 1px; 
+    margin-top: calc((1rem - 12px) / 2);
```

Comment:
This looks fine on my screen, but instead we should use `align-items: center` on `.section-label-row` so that css deals with the calculation for us (especially if we change the font size for some reason)

Reply:
This is literally what flex was made for, nice catch